dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -405,8 +407,27 @@ object LagomPlugin extends AutoPlugin {
   }
 
   private def runAllMicroservicesTask: Initialize[Task[Unit]] = Def.taskDyn {
+
     val projects = microservicesProjects.value
-    val filter = ScopeFilter(inProjects(projects: _*))
+
+    // check if application.conf is present in service
+    def isValidProject(p: ProjectRef) = {
+
+      val prj = Project.getProject(p, buildStructure.value)
+
+      val r: PathFinder = {
+        (new File(prj.get.base.getPath) / "src") ** "application.conf"
```

Comment:
Using the Sbt [resourceDirectories](http://www.scala-sbt.org/0.13/sxr/sbt/Keys.scala.html#sbt.Keys.resourceDirectories) key may be preferable.

However, if we use `resourceDirectories`, we are going to prevent people from placing their application.conf in a source directory. I think that's actually a good thing, as we are opinionated and we want to push people into using best practices.

Also, if I'm not mistaken, the application.conf file should be placed in an immediate children of the directory (hence, I think we should be using `Path#*` instead of `Path#**`, but this should be double checked).